DESIGN.md

```diff
@@ -48,11 +50,12 @@ to three strong requirements:
    into account can easily burn through blocks used to store frequently updated
    metadata.
 
-   Consider the [FAT filesystem](https://en.wikipedia.org/wiki/Design_of_the_FAT_file_system),
+     Consider the [FAT filesystem](https://en.wikipedia.org/wiki/Design_of_the_FAT_file_system),
```

Comment:
Out of curiousity, why the indention on these paragraphs?

Reply:
Without the (ugly) indentation retext(1) would not render this properly, fwiw. Do you want me to drop this hunk?

Reply:
Well if you give me a choice I'd prefer to not indent : )

I checked with a bunch of online markdown renderers and it looks like this might be an issue with retext specifically.